suricata/update/sources.py

```diff
@@ -130,6 +130,12 @@ def get_source_by_name(self, name):
             return self.index["sources"][name]
         return None
 
+    def get_versions(self):
+        try:
+            return self.index["versions"]
+        except KeyError:
+            raise Exception("Version information not in index")
```

Comment:
Gently catch this exception and log an error? This gives an ugly traceback, maybe we should do it better.